README.md

```diff
@@ -696,60 +631,60 @@ Chocolatey v0.9.9.0+.
 #### Properties/Parameters
 
 ##### `name`
-(**Namevar**: If ommitted, this attribute's value will default to the resource's
-title.)
 
-The name of the source. Used for uniqueness. Will set the location to this value if location is unset.
+Specifies the name of the source. Used for uniqueness. Also sets the `location` to this value if `location` is unset. Valid options: String. Default: The resource's title.
 
 ##### `ensure`
-(**Property**: This attribute represents concrete state on the target system.)
 
-What state the source should be in. This defaults to `present`.
+(**Property**: This parameter represents a concrete state on the target system.)
 
-Valid options: `present`, `disabled`, or `absent`.
+Specifies what state the source should be in. Default: `present`. Valid options: `present`, `disabled`, or `absent`.
 
 ##### `location`
-(**Property**: This attribute represents concrete state on the target system.)
 
-The location of the source repository. Can be a url pointing to an OData feed (like chocolatey/chocolatey_server), a CIFS (UNC) share, or a local folder. Defaults to the name of the resource.
+(**Property**: This parameter represents a concrete state on the target system.)
+
+Specifies the location of the source repository. Valid options: String of a URL pointing to an OData feed (such as chocolatey/chocolatey_server), a CIFS (UNC) share, or a local folder. Default: The `name` of the resource.
 
 ##### `user`
-(**Property**: This attribute represents concrete state on the target system.)
 
-Optional user name for authenticated feeds. Requires at least Chocolatey v0.9.9.0. Defaults to `nil`. Specifying an empty value is the same as setting the value to nil or not specifying the property at all.
+(**Property**: This parameter represents a concrete state on the target system.)
+
+Specifies an optional user name for authenticated feeds. Requires at least Chocolatey v0.9.9.0. Default: `nil`. Specifying an empty value is the same as setting the value to `nil` or not specifying the property at all.
 
 ##### `password`
-Optional user password for authenticated feeds. Not ensurable. Value is not able to be checked with current value. If you need to update the password, update another setting as well. Requires at least Chocolatey v0.9.9.0. Defaults to `nil`. Specifying an empty value is the same as setting the value to nil or not specifying the property at all.
+
+Specifies an optional user password for authenticated feeds. Not ensurable. Value is not able to be checked with current value. If you need to update the password, update another setting as well. Requires at least Chocolatey v0.9.9.0. Default: `nil`. Specifying an empty value is the same as setting the value to `nil` or not specifying the property at all.
```

Comment:
Value cannot be checked with current value.

Also, I'd add something to explain this sentence:

"If you need to update the password, update another setting as well." Maybe just add "...to force the update." (assuming I'm understanding this).